synse/factory.py

```diff
@@ -72,6 +76,14 @@ def make_app():
     return app
 
 
+async def periodic_cache_invalidation():
+    """Periodically invalidate the caches so they are rebuilt."""
+    while True:
+        await asyncio.sleep(5 * 60)  # 5 minutes
+        logger.info('task [periodic cache invalidation]: clearing device caches')
+        await clear_all_meta_caches()
```

Comment:
If this method raises an exception the background job will terminate. I'd be good to capture, filter, and bubble/log appropriately